gossip/privdata/coordinator.go

```diff
@@ -407,63 +409,51 @@ func getTxInfoFromTransactionBytes(envBytes []byte) (*txInfo, error) {
 	txInfo := &txInfo{}
 	env, err := protoutil.GetEnvelopeFromBlock(envBytes)
 	if err != nil {
-		logger.Warningf("Invalid envelope: %s", err)
-		return nil, err
+		return nil, fmt.Errorf("invalid envelope: %s", err)
 	}
 
 	payload, err := protoutil.UnmarshalPayload(env.Payload)
 	if err != nil {
-		logger.Warningf("Invalid payload: %s", err)
```

Comment:
I don't understand the motivation behind this change at all. All these errors are not things that should appear during normal operation. Why are they logged at debug? 

The JIRA details a single warning about a transaction not being an endorser type. 

Why shouldn't all the other errors be logged with warnings?

Reply:
My understanding was the warning logs in this specific path seem extraneous to show up during regular gossip flow, especially since we skip the transactions that we fail to retrieve private data from anyways. The thought here being it's a debug statement to mention we're skipping a particular transaction in the block. I'm indifferent though and can understand leaving the other lines at warning.

Reply:
> The thought here being it's a debug statement to mention we're skipping a particular transaction in the block. 

My problem with it, is that we are skipping the transaction due to something that should never happen under normal operations. If you have transactions that have malformed fields inside of them, you would want to know that and tell it to someone or ask someone even if you aren't effected by it, because someone else is.